### ISSUE TITLE
Pattern "<<.+?>>" causes side effects

### ISSUE THREAD

**Issue by swiehr** [maphy-psd/dokuwiki-plugin-date#2]
The pattern registration for "<<.+?>>" and its handling causes strange side effects on pages where entity replacement and quotation syntax is used as it behaves quite greedy.
Why register this pattern anyway? It seems not be needed for the plugin.

**Comment by maphy-psd** [comment]
This pattern is used for the constants. <<today>> gives you today's date.
Maybe I can't change the pattern to {{date>constant}} 

Have you an example site or can you describe a situation for your strange side effects?

**Comment by swiehr** [comment]
Take this snippet and see how it renders differently with or without the pattern:

```
-> <- <-> => <= <=> >> << -- --- 640x480 (c) (tm) (r)
"He thought 'It's a man's world'..."

The same can be done to produce any kind of HTML, it just needs to be added to the [[doku>entities|pattern file]].

There are three exceptions which do not come from that pattern file: multiplication entity (640x480), 'single' and "double quotes". They can be turned off through a [[doku>config:typography|config option]].

===== Quoting =====

Some times you want to mark some text to show it's a reply or comment. You can use the following syntax:

I think we should do it

> No we shouldn't

>> Well, I say we should

> Really?

>> Yes!

>>> Then lets do it!
```

**Comment by maphy-psd** [comment]
Ok, i will change the syntax for the constant. I think {{date>constant}} is useful.

**Comment by maphy-psd** [comment]
fix by 3599cfd - close
